app/controllers/stripe-setup/bank-details/post.controller.js

```diff
@@ -35,20 +37,34 @@ module.exports = async (req, res, next) => {
     return response(req, res, 'stripe-setup/bank-details/index', {
       accountNumber: rawAccountNumber,
       sortCode: rawSortCode,
+      switchingToCredentials,
       errors
     })
   }
 
   try {
-    const stripeAccount = await connector.getStripeAccount(req.account.gateway_account_id, req.correlationId)
+    const switchingCredential = getSwitchingCredentialIfExists(req.account)
+    let stripeAccountId
 
-    await updateBankAccount(stripeAccount.stripeAccountId, {
+    if (switchingToCredentials) {
```

Comment:
`switchingToCredentials` is quite similar to `switchingCredential`
maybe renaming to something like `isSwitchingCredentials` would improve readability